README.md

```diff
@@ -88,6 +87,11 @@ to use, test, and provide feedback on the new features are encouraged to try
 this branch. However, this branch provides no stability guarantees and reserves
 the right to change and break the API at any time.
 
+##Prerequisite knowledge
+
+Before attempting to understand this framework one should ideally have some familiarity with the structure and format of the Google Test framework, upon which it is based. Documentation for Google Test, including a "Getting Started" (primer) guide, is available here:
```

Comment:
But this is not *based* on that. They just happen to have a similar approach. As far as i know.

Reply:
This is actually part of a previous commit (see current readme.md). Google benchmark is indeed based on the gtest framework - that's why you need it to compile it.

Reply:
Uhm, no?
Can you quote the test you think says that?

Reply:
What?
Have you even compiled google benchmark yourself before?
At any rate, it's part of a previous commit which has Already Been Accepted.

Reply:
Uhm, if that is part of the README.md as of current git master, then why/how it is in this diff?
Since that line was added by #600 by You, the question remains unanswered.

Reply:
It was answered by the commit itself. Look it up kid.

Reply:
Maybe i just can't read. Which one?
* https://github.com/google/benchmark/pull/600/commits/53786ab39e68da9f9c4aeba0f9177fd921b37f6a
  > Third option under this  line: "Note that Google Benchmark requires Google Test to build and run the tests. This dependency can be provided three ways:"
  > Was not true (did not occur). If there is a further option that needs to be specified in order for that functionality to work it needs to be specified.

  That is not correct, it's clearly occurring - https://github.com/google/benchmark/blob/6d74c0625b8e88c1afce72b4f383c91b9a99dbe6/cmake/HandleGTest.cmake#L102
* https://github.com/google/benchmark/pull/600/commits/cf9e96815b298643c99831554e17a1ad6a25bd54 commit msg doesn't actually answer it, it just states it as a fact.

Reply:
You're not being helpful, or posting under the right heading, and I'm going to ignore you.